qubes/tests/api_internal.py

```diff
@@ -160,6 +162,7 @@ def test_010_get_system_info(self):
         vm.label.icon = 'icon-vm'
         vm.guivm = vm
         vm.get_power_state.return_value = 'Halted'
+        vm.uuid = uuid.uuid4()
```

Comment:
I'd prefer specific hardcoded value in the test (simply generate new random one) instead of ever changing random here.